src/components/Card/__snapshots__/Card.story.storyshot

```diff
@@ -560,17 +577,6 @@ exports[`Storybook Snapshot tests and console checks Storyshots Watson IoT/Card
                   Cancel
                 </button>
               </div>
-              <div
```

Comment:
Is it intentional that this moved? Looks like it moved in front of the cancel button

Reply:
Yep https://github.com/carbon-design-system/carbon/pull/6566

Though I've uncovered a bug with batch actions thanks to looking into this. Will push a fix.